src/reconcile/rec_row.c

```diff
@@ -621,9 +621,6 @@ __rec_row_leaf_insert(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins)
             ret = __wt_illegal_value(session, upd->type);
             WT_ERR(ret);
         }
-        /* Free the update if it is external. */
```

Comment:
@sulabhM We talked about this earlier but I don't think this is quite right. This is within a loop and I think only the last one should be kept around.